src/components/editor/property/InputLiteral.jsx

```diff
@@ -118,13 +119,13 @@ export class InputLiteral extends Component {
     this.inputLiteralRef.current.focus()
   }
 
+  /**
+   * @return {bool} true if the field should be marked as required (e.g. not all obligations met)
```

Comment:
say default is false?  or is that obvious?

Reply:
I think it's pretty clear from how it's documented that if it doesn't meet that condition it's false.